flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
since you build from source any reason to limit the arch?

Reply:
As far as I know, the official build (and also the Snap package) is x86_64 only. Also, I would have to modify the startup script to detect arch and run the AARCH64 binaryversion if on AARCH64

Reply:
As far as I know, the official build (and also the Snap package) is x86_64 only. Also, I would have to modify the startup script to detect arch and run the AARCH64 binary if on AARCH64. I plan to do it later when it is properly tested and confirmed that it actually works on AARCH64 (which I am unable to do at the moment since I now have only x86_64 machines here).

Reply:
`$FLATPAK_ARCH` is here for that.

Reply:
Oops, another thing I forgot about. :-) Thanks a lot!

Reply:
```
[ 48%] Generate spirv for shaders/single_texture.vert (vert): single_texture_vert.vspv
/bin/sh: line 1: /run/build/wop/code/../tools/linux/glslangValidator: cannot execute binary file: Exec format error
```

It does not build on aarch64 since the _glslangValidator_ binary used to generate spirv seems to be x86_64-only.

Reply:
executing a random binary at build time. nothing can go wrong.... really.